package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "zipkin-api",
+  "version": "0.0.0",
+  "description": "Zipkin API ",
+  "main": "validate.js",
+  "scripts": {
+    "test": "node validate.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openzipkin/zipkin-api.git"
+  },
+  "keywords": [
+    "zipkin",
+    "api"
+  ],
+  "author": "",
+  "license": "ISC",
```

Comment:
ok, if we aren't publishing to an npm repo, let's switch this to ASL, or just drop the key, if you don't mind. I'd like the license surface area of zipkin to be as small as possible. In other words, let's choose our battles.